content/en/event-streaming.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Event Streaming
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+
+Event streaming is an approach where software sends event data from one application to another to continuously communicate what they are doing.
+Picture a service broadcasting everything it does to all other services.
+Each activity taken by a service is referred to as an event, hence event streaming.
+For example, the NASDAQ gets updates on stock and commodities pricing every second. 
+An application that monitors a specific set of stocks would need to receive that information in near real-time. 
+If you had an application that monitored a specific set of stocks, you would want to receive that information in near real-time. 
+If NASDAQ provides an API that send the information from their application to yours, you have event streaming.
+The data sent and the changes in that data (stock prices) are the events, while the process of delivering them to an application is event streaming.
+
+## Problem it addresses
+
+Traditionally, one would use single TCP requests. 
```

Comment:
```suggestion
Traditionally, NASDAQ would have provided this data via TCP requests.
```
Since we are referring to the example above, we should refer to it in the new section